sdk/textanalytics/cognitiveservices-textanalytics/README.md

```diff
@@ -21,30 +28,89 @@ Use the client library to:
 
 - Node.js version 8.x.x or higher
 
-**Prerequisites**: You must have an [Azure Subscription](https://azure.microsoft.com) and an [Cognitive Services](https://docs.microsoft.com/en-us/azure/cognitive-services/cognitive-services-apis-create-account) resource to use this package.
+### Prerequisites
+
+- An [Azure subscription][azure_sub].
+- An existing [Cognitive Services][cognitive_resource] or Text Analytics resource. If you need to create the resource, you can use the [Azure Portal][azure_portal] or [Azure CLI][azure_cli].
+
+If you use the Azure CLI, replace `<your-resource-group-name>` and `<your-resource-name>` with your own unique names:
+
+```PowerShell
+az cognitiveservices account create --kind TextAnalytics --resource-group <your-resource-group-name> --name <your-resource-name>
+```
 
 ### 1. Install the `@azure/cognitiveservices-textanalytics` package
 
 ```bash
-npm install @azure/cognitiveservices-textanalytics
+npm install --save @azure/cognitiveservices-textanalytics
```

Comment:
We don't have the `--save` in any of our other Track 2 libraries AFAIK
If this is a standard we want to follow, please log an issue to update the rest.